src/rulesets/Base/arraymath.jl

```diff
@@ -268,3 +268,15 @@ function rrule(::typeof(-), x::AbstractArray)
     end
     return -x, negation_pullback
 end
+
+
+#####
+##### Addition (Multiarg `+`)
+#####
+
+function rrule(::typeof(+), arrs::AbstractArray...)
+    y = +(arrs...)
+    valN = Val(length(arrs)) # makes add_pullback type-stable without closing over arrs
+    add_pullback(dy) = (NoTangent(), ntuple(_ -> dy, valN)...)
+    return y, add_pullback
+end
```

Comment:
This is a version which would remove trailing dimensions
```suggestion
function rrule(::typeof(+), arrs::AbstractArray...)
    y = +(arrs...)
    arr_axs = map(axes, arrs)
    function add_pullback(dy_raw)
        dy = unthunk(dy_raw)
        return (NoTangent(), map(ax -> reshape(dy, ax), arr_axs)...)
    end
    return y, add_pullback
end
```